src/wazuh.py

```diff
@@ -353,3 +354,57 @@ def _generate_cluster_snippet(
             <disabled>no</disabled>
         </cluster>
     """
+
+
+def change_api_password(username: str, old_password: str, new_password: str) -> None:
+    """Change Wazuh's API password for the default 'wazuh' user.
```

Comment:
Description is misleading, it's not only for the 'wazuh' user but for the given user